pkgs/top-level/all-packages.nix

```diff
@@ -17190,7 +17190,9 @@ with pkgs;
   wlroots = callPackage ../development/libraries/wlroots { };
   rootston = wlroots.bin;
   orbment = callPackage ../applications/window-managers/orbment { };
-  sway = callPackage ../applications/window-managers/sway { };
+  swayPkgs = callPackage ../applications/window-managers/sway { };
+
+  inherit (swayPkgs) sway sway-beta;
```

Comment:
Must be updated (sway + sway-beta).

Reply:
Oh, whoops, forgot to check that in :sweat_smile: